### DISCUSSION TITLE
A question on editing the mod and removing particular files.

### DISCUSSION THREAD

**Discussion by Xinvolakh** [acemod/ACE3#8314] [Q&A] [answered]
I desperately want to use this mod without advanced ballistics, as it ruins the game entirely for me. In short; I hate this mod because I can't snipe and use it at the same time. I've searched everywhere, I've deleted the pbo and all it does is corrupt the file. I've tried everything I know to, and I still can't figure it out. Help please lol.

**Comment by Drofseh** [comment, marked as answer]
Advanced Ballistics can be turned off in your Addon Options settings, no need to delete pbos.
(Also, questions are best asked in the Ace3 Slack. <https://slackin.ace3mod.com/>)

![](https://i.gyazo.com/aa61bfb3f5218e0d564116cfb42e43f3.png)
